### PR TITLE
Improve convert via to 3d

### PR DESCRIPTION
**Pull request by amichel0205**
Improve convert via to 3D.
Add the ability to not remove the padstack definition and to set hole size to zero.

### CODE REVIEW

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 1]
```diff
@@ -739,6 +739,9 @@ def convert_to_3d_microvias(self, convert_only_signal_vias=True, hole_wall_angle
             HoleDiameter -2*tan(laser_angle* Hole depth). Hole depth is the height of the via (dielectric thickness).
             The default value is ``15``.
             The bottom hole will be ``0.75*HoleDepth/HoleDiam``.
+        delete_padstack_def: bool, optional
+            Either to remove or not padstack definition. Defaults is ``True``.
+            When set to False padstack definition is not removed and hole size is set to zero.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:744]:
```suggestion
            Whether to delete the padstack definition. The default is ``True``.
            If ``False`` the padstack definition is not deleted and the hole size is set to zero.
```

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 2]
```diff
@@ -739,6 +739,9 @@ def convert_to_3d_microvias(self, convert_only_signal_vias=True, hole_wall_angle
             HoleDiameter -2*tan(laser_angle* Hole depth). Hole depth is the height of the via (dielectric thickness).
             The default value is ``15``.
             The bottom hole will be ``0.75*HoleDepth/HoleDiam``.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:741]:
```suggestion
            The default is ``15``.
            The bottom hole is ``0.75*HoleDepth/HoleDiam``.
```

**Comment by amichel0205** [comment]
Could you check my last commit it seems there is a modification on file Design.py but it is not coming from me.
I've merged updated main in my branch before to push.

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 3]
```diff
@@ -737,13 +737,21 @@ def convert_to_3d_microvias(self, convert_only_signal_vias=True, hole_wall_angle
         hole_wall_angle : float, optional
             Angle of laser penetration in deg. It will define the bottom hole diameter with the following formula:
             HoleDiameter -2*tan(laser_angle* Hole depth). Hole depth is the height of the via (dielectric thickness).
-            The default value is ``15``.
-            The bottom hole will be ``0.75*HoleDepth/HoleDiam``.
+            The default is ``15``.
+            The bottom hole is ``0.75*HoleDepth/HoleDiam``.
+        delete_padstack_def: bool, optional
+            Whether to delete the padstack definition. The default is ``True``.
+            If ``False`` the padstack definition is not deleted and the hole size is set to zero.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:744]:
```suggestion
            If ``False``, the padstack definition is not deleted and the hole size is set to zero.
```

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 4]
```diff
@@ -737,13 +737,21 @@ def convert_to_3d_microvias(self, convert_only_signal_vias=True, hole_wall_angle
         hole_wall_angle : float, optional
             Angle of laser penetration in deg. It will define the bottom hole diameter with the following formula:
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:738]:
```suggestion
            Angle of laser penetration in degrees. The angle defines the bottom hole diameter with this formula:
```

File: pyaedt/edb_core/edb_data/padstacks_data.py (hunk 3 above)
Comment (amichel0205) [reply pyaedt/edb_core/edb_data/padstacks_data.py]: Suggested change applied

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Pull request merged**